core/chains/evm/forwarders/forwarder.go

```diff
@@ -0,0 +1,17 @@
+package forwarders
+
+import (
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/smartcontractkit/chainlink/core/utils"
+)
+
+// Forwarder is the external interface for ForwarderAddresses
+type EVMForwarder struct {
```

Comment:
Or, since we are are in `package forwarders`, just: `forwarders.Address`?

Reply:
I agree for renaming to Forwarder (I was trying to reinforce that this is only evm compatible for now) but maybe forwarders.Forwarder as I changed it is more canonical to golang? happy to change

Reply:
nbd but yeah I was thinking the idiomatic go approach would be to consider the `EVM` prefix redundant due to the directory context, and also to avoid the stuttering/redundancy of `forwarders.Forwarder`.
https://go.dev/doc/effective_go#names
> The importer of a package will use the name to refer to its contents, so exported names in the package can use that fact to avoid repetition. (Don't use the import . notation, which can simplify tests that must run outside the package they are testing, but should otherwise be avoided.) For instance, the buffered reader type in the bufio package is called Reader, not BufReader, because users see it as bufio.Reader, which is a clear, concise name. Moreover, because imported entities are always addressed with their package name, bufio.Reader does not conflict with io.Reader. Similarly, the function to make new instances of ring.Ring—which is the definition of a constructor in Go—would normally be called NewRing, but since Ring is the only type exported by the package, and since the package is called ring, it's called just New, which clients of the package see as ring.New. Use the package structure to help you choose good names.

Reply:
That's insightful. Funny that golang itself doesn't adhere to that. I guess I'll send out another clean up for this for you to review then